contents/blog/new-pricing.md

```diff
@@ -0,0 +1,148 @@
+---
+date: 2024-06-24
+title: 'PostHog is now up to 80% cheaper'
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog_is_the_cheapest_e77c4ea4a5.jpg
+author:
+  - james-hawkins
+tags:
+  - PostHog news
+---
+import {ProductScreenshot} from 'components/ProductScreenshot'
+
+export const Pricing = "https://res.cloudinary.com/dmukukwp6/image/upload/posthog_pricing_ba910c9535.jpg"
+export const PricingDark = "https://res.cloudinary.com/dmukukwp6/image/upload/posthog_pricing_dark_6b11387d6c.jpg"
+export const PricingComparison = "https://res.cloudinary.com/dmukukwp6/image/upload/posthog_pricing_comparison_f0c9e11f40.jpg"
+export const PricingComparisonDark = "https://res.cloudinary.com/dmukukwp6/image/upload/posthog_pricing_comparison_dark_ecf7d726c0.jpg"
+
+## TL;DR
+
+* You can now use our product and web analytics for up to 80% less, making us cheaper than any popular competitive product.
+
+* This is possible because we've introduced a new event processing option that enables you to capture events without person data. 
+
+* Events without person data are cheaper for us to ingest and query. We're passing this saving onto you by introducing cheaper base event pricing and a new 'person profiles' event option, which is charged separately.
+
+* Events with person profiles enabled cost the same as our old event pricing – i.e. base events + person profiles = our previous per event pricing. If you're an existing customer, you'll see a new 'person profiles' line item on your invoice, but your pricing has not increased.
+
+* By default, events are captured with person profiles, but you can change this with a simple [configuration option](#how-do-i-start-using-it) whenever you like.
```

Comment:
@joethreepwood Added jump link to config section of this page as I'd rather people not leave the page at this stage.